modules/authorization/role-definitions/resource-group/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-metadata-file-schema-v2#",
+  "name": "Role Definitions on Resource Group level",
+  "summary": "With this module you can create role definitions on a resource group level",
```

Comment:
Should the README.md be updated to "This module deploys Role Definitions at a Resource Group level." and then this file updated accordingly?